test/one_time_password_test.rb

```diff
@@ -13,6 +13,10 @@ def setup
     @member = Member.new
     @member.email = nil
     @member.run_callbacks :create
+
+    @ar_user = ActiverecordUser.new
+    @ar_user.email = 'roberto@heapsource.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.